vscode-bebop/src/extension.ts

```diff
@@ -46,12 +47,41 @@ export function activate(context: vscode.ExtensionContext) {
     const client = new lsp.LanguageClient('bebopLanguageServer', 'Bebop Language Server', serverOptions, clientOptions);
     client.trace = Trace.Verbose;
     let disposable = client.start();
-	vscode.window.showInformationMessage(`Bebop language server started (${serverExe})`);
+	vscode.window.showInformationMessage(`Bebop language server started (${executable})`);
 
     // Push the disposable to the context's subscriptions so that the
     // client can be deactivated on extension deactivation
     context.subscriptions.push(disposable);
 }
 
+function getBebopCompilerPath(context: vscode.ExtensionContext) {
+    // Got an environment variable?
+    let envPath = process.env.BEBOP_LANGUAGE_SERVER_PATH;
+    if(envPath !== undefined) {
+        return envPath;
+    }
+
+    // In development mode?
+    if (context.extensionMode === vscode.ExtensionMode.Development) {
+
+        let os = platform();
+        if (os === "win32") {
+            return path.resolve(path.join(context.extensionPath, '../bin/compiler/Windows-Debug/bebopc.exe'));
+        } else {
+            return path.resolve(path.join(context.extensionPath, '../bin/compiler/Linux-Debug/bebopc.exe'));
+        }
+    }
+
+    // Use the packaged compiler
```

Comment:
bebopc ships for macOS, Linux, and Windows so we’ll need to update the appropriate pathing.